src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/dwpuploadresponse/DwpUploadResponseAboutToSubmitHandlerTest.java

```diff
@@ -1130,64 +1165,163 @@ public void givenOtherPartiesUcbIsYes_thenUpdateCasedataOtherPartyUcb() {
 
     @Test
     public void givenNewOtherPartyAdded_thenAssignAnId() {
-        sscsCaseData.setOtherParties(Arrays.asList(buildOtherPartyWithAppointeeAndRep(null, null, null)));
+        sscsCaseData.setOtherParties(Arrays.asList(
+            buildOtherPartyWithAppointeeAndRep(null, null, null)));
         sscsCaseData.getAppeal().getBenefitType().setCode("childSupport");
 
         PreSubmitCallbackResponse<SscsCaseData> response = dwpUploadResponseAboutToSubmitHandler.handle(ABOUT_TO_SUBMIT, callback, USER_AUTHORISATION);
-        assertEquals(1, response.getData().getOtherParties().size());
-        assertEquals("1", response.getData().getOtherParties().get(0).getValue().getId());
-        assertEquals("2", response.getData().getOtherParties().get(0).getValue().getAppointee().getId());
-        assertEquals("3", response.getData().getOtherParties().get(0).getValue().getRep().getId());
-        assertTrue(YesNo.isYes(response.getData().getOtherParties().get(0).getValue().getSendNewOtherPartyNotification()));
+
+        Assertions.assertThat(response.getData().getOtherParties())
+            .hasSize(1)
+            .extracting(CcdValue::getValue)
+            .extracting(OtherParty::getSendNewOtherPartyNotification)
+            .containsOnly(YES);
+
+        Assertions.assertThat(response.getData().getOtherParties())
+            .extracting(CcdValue::getValue)
+            .extracting(Entity::getId)
+            .hasSize(1)
+            .doesNotContainNull()
+            .extracting(String::length)
+            .containsOnly(36);
+
+        Assertions.assertThat(response.getData().getOtherParties())
+            .extracting(CcdValue::getValue)
+            .flatExtracting(otherParty -> List.of(otherParty.getAppointee(), otherParty.getRep()))
+            .extracting(Entity::getId)
+            .hasSize(2)
+            .doesNotContainNull()
+            .extracting(String::length)
+            .containsOnly(36);
     }
 
     @Test
     public void givenExistingOtherParties_thenNewOtherPartyAssignedNextId() {
```

Comment:
I don't think we need tests which regard assigning next ids since we're switching to UUIDs

Reply:
Renamed the tests